contrib/zellij/template.py

```diff
@@ -0,0 +1,35 @@
+pkgname = "zellij"
+pkgver = "0.40.1"
+pkgrel = 0
+build_style = "cargo"
+make_check_args = ["--release"]
+hostmakedepends = ["cargo-auditable", "pkgconf", "openssl-devel", "go-md2man"]
+pkgdesc = "Terminal workspace with batteries included"
+maintainer = "Denis Strizhkin <strdenis02@gmail.com>"
+license = "MIT"
+url = "https://github.com/zellij-org/zellij"
+source = f"{url}/archive/refs/tags/v{pkgver}.tar.gz"
+sha256 = "1f0bfa13f2dbe657d76341a196f98a3b4caa47ac63abee06b39883a11ca220a8"
+
+def post_build(self):
+    with open(self.cwd / "docs/MANPAGE.md", "rb") as i:
+        with open(self.cwd / f"docs/{pkgname}.1", "w") as o:
```

Comment:
```suggestion
        with open(self.cwd / f"docs/zellij.1", "w") as o:
```

here and everywhere else, generally always want to expand pkgname